DLATK support for Google Colab

shshnk94 opened the pull request:
In addition to the post-installation shell script and changes in `dlatkInterface.py`, this branch also includes -

- Changes in `setup.py` to place `colabify.sh` inside `{DLATK_PATH}`.
- Initial draft of the documentation for the `-colabify` flag.

shshnk94 committed:
DLATK for Google Colab

shshnk94 committed:
Added doc for --colabify

shshnk94 committed:
removed an open pdb trace

gtsherman commented:
This looks excellent to me. My only concern is that I'm not sure the main `dlatk` directory is the best location for `colabify.sh`. How about moving it to `dlatk/tools/` where we have other scripts? Other than that one change, I'm happy with this (assuming it has been tested and all works well).

shshnk94 committed:
moved colabify.sh to dlatk/tools as per PR comments

shshnk94 committed:
answers prompt in colabify.sh

shshnk94 committed:
installs py3.6 if not found, then redirects

shshnk94 commented:
### Changes

- Moved `colabify.sh` to `dlatk/tools` as per the above suggestion. 
- Added `-y` flag to `apt-get`s.
- Installs Python 3.6 if not found in `/usr/bin`.
- `--force` in `update-alternatives` to override existing `/usr/local/bin/python`.

The above changes were tested in [this notebook](https://drive.google.com/file/d/1Ffe4F85HkmxRbeHD9sL31R_vipq9akRA/view?usp=sharing).

gtsherman commented:
Looks perfect to me! Thanks for your hard work on this.
